src/Api/Utilities/ExceptionHandlerFilterAttribute.cs

```diff
@@ -92,6 +92,11 @@ public override void OnException(ExceptionContext context)
             errorMessage = "Unauthorized.";
             context.HttpContext.Response.StatusCode = 401;
         }
+        else if (exception is AggregateException aggregateException)
+        {
+            errorMessage = string.Join(Environment.NewLine, aggregateException.InnerExceptions.Select(ex => ex.Message));
```

Comment:
`Environment.NewLine` doesn't work here because it's not rendered properly in HTML. Try throwing an error in the invite users loop and then invite multiple users (to cause multiple errors) - you'll see that the error messages are all run together. HTML doesn't recognise newline chars.

I think there's a better way to do this. `ErrorResponseModel` [supports multiple error messages](https://github.com/bitwarden/server/blob/d5d49a6cf146358f1a1207c8aba92630982b4a24/src/Api/Models/Public/Response/ErrorResponseModel.cs#L71-L75). This is supported client-side by [ValidationService](https://github.com/bitwarden/clients/blob/master/libs/common/src/services/validation.service.ts#L21) and showToast will [compose it into multiple lines for you](https://github.com/bitwarden/clients/blob/master/apps/web/src/app/app.component.ts#L281). So if you just send all the error messages back to the client, it _should_ just work.

Reply:
That was excellent advice! I've changed it to use `ErrorResponseModel`'s multiple error messages and it worked perfectly.